h/views/admin/users.py

```diff
@@ -18,6 +18,9 @@ class UserNotFoundError(Exception):
 def format_date(date):
     """Format a date for presentation in the UI."""
 
+    if date is None:
```

Comment:
I think pylint might say `if not date` if we had pylint in h

Reply:
I'm specifically trying to distinguish between an empty string (which actually should fail) and NULL from the DB here, so I think `is None` is the right way.